### DISCUSSION TITLE
Topic "Selenium-Standalone-Service" is deprecated or just documentation is broken?

### DISCUSSION THREAD

**Discussion by danielsvelicenko** [webdriverio/webdriverio#11568] [Q&A]
I remember that there was some documentation on Selenium Standalone service for WebdriverIO. It's bit confusing, that we can search for it, but then display that can't find anything.

# Steps:
1. Search for the "Selenium Standalone Service"
![image](https://github.com/webdriverio/webdriverio/assets/81615057/2702447f-2e39-4518-b58a-b0cec4654f7b)
2. Select any sub-topic that goes under it

# Actual results:
![image](https://github.com/webdriverio/webdriverio/assets/81615057/152fb94f-2f78-4a49-b6cc-59a64b5a3cb4)

**Comment by danielsvelicenko** [comment]
And somewhy, re-opening via link, it's shows broken UI there.
https://webdriver.io/docs/selenium-standalone-service/

![image](https://github.com/webdriverio/webdriverio/assets/81615057/6d8a5d6d-16e3-4558-aaa9-d7c8860d6374)

**Comment by christian-bromann** [comment]
Thanks for reporting!

I think we should clear up the website here and remove these references and pages. I can look into this. This happens because we never really clear the S3 bucket when we re-redeploy so removed pages stick around.
